src-backend/api/tests/test_pages.py

```diff
@@ -0,0 +1,107 @@
+from django.test import TestCase, Client, SimpleTestCase
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+from nose.tools import assert_equals
+from api.models import User, Page, Procedure
+from utils.decorators import add_group_permissions
+import json
+
+
+class PageTest(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.user = User.objects.create_user(
+                    'admin',
+                    'test@email.com',
+                    'password'
+                )
+        self.procedure = Procedure.objects.create(
+                    author='tester',
+                    title='test procedure',
+                    owner=self.user
+                )
+        self.page1 = Page.objects.create(
+                    display_index=0,
+                    procedure=self.procedure
+                )
+        self.page2 = Page.objects.create(
+                    display_index=1,
+                    procedure=self.procedure
+                )
+        self.page3 = Page.objects.create(
+                    display_index=2,
+                    procedure=self.procedure
+                )
+
+    @add_group_permissions
+    def test_page_reorder_back(self):
+        data = [
```

Comment:
Different indices, not sure if we really need to do that. I tried using a json serializer, but it only worked on objects returned by `Page.objects.find(...)` so it had to be data already stored in the database. Modifying that would be a lot uglier and more work than simply defining the data as this